srcalc/src/main/java/gov/va/med/srcalc/vista/vistalink/VistaLinkUtil.java

```diff
@@ -0,0 +1,35 @@
+package gov.va.med.srcalc.vista.vistalink;
+
+import gov.va.med.vistalink.institution.InstitutionMapNotInitializedException;
+import gov.va.med.vistalink.institution.InstitutionMappingDelegate;
+import gov.va.med.vistalink.institution.InstitutionMappingNotFoundException;
+
+/**
+ * Provides VistALink utility methods. These are not srcalc-specific: VistALink could have
+ * provided them itself.
+ */
+public class VistaLinkUtil
+{
+    private VistaLinkUtil()
+    {
+        // No construction.
+    }
+
+    /**
+     * Returns true if the given division is known to VistALink, false otherwise.
+     */
+    public static boolean isDivisionKnown(final String division)
+    {
+        try
+        {
+            InstitutionMappingDelegate.getJndiConnectorNameForInstitution(division);
+        }
+        catch (InstitutionMapNotInitializedException | InstitutionMappingNotFoundException e)
```

Comment:
I have literally never seen this. Did not even know you could catch multiple Exceptions like this.